code/game/striketeams/striketeam_datums.dm

```diff
@@ -0,0 +1,185 @@
+var/list/sent_strike_teams = list()
+
+/datum/striketeam
+	var/striketeam_name = "Spec.Ops."
+	var/faction_name = "Nanotrasen"
+	var/mission = "Clean up the Station of all enemies of Nanotrasen. Avoid damage to Nanotrasen assets, unless you judge it necessary."
+	var/team_size = 6
+	var/min_size_for_leader = 4//set to 0 so there's always a designated team leader or to -1 so there is no leader.
+	var/spawns_name = "Commando"
+	var/can_customize = FALSE
+	var/logo = "nano-logo"
+
+	var/list/applicants = list()
+	var/searching = FALSE
+
+	var/leader_key = ""
+	var/list/team_composition = list()
+
+
+/datum/striketeam/proc/trigger_strike(var/mob/user)
+	//Is the game started
+	if(!ticker)
```

Comment:
Ticker existing does not necessarily entail the game having started.

The ticker exists as soon as the MC begins set up, finishes pre-round init when the MC is done, but this is all before the lobby timer even begins to tick down.